src/internal/components/series-marker/index.tsx

```diff
@@ -21,16 +22,17 @@ export interface ChartSeriesMarkerProps extends BaseComponentProps {
   type: ChartSeriesMarkerType;
   color: string;
   visible?: boolean;
+  status?: "warning";
 }
 
 function scale(size: number, value: number) {
   return `translate(${size * ((1 - value) / 2)}, ${size * ((1 - value) / 2)}) scale(${value})`;
 }
 
-export function ChartSeriesMarker({ type = "line", color, visible = true }: ChartSeriesMarkerProps) {
+export function ChartSeriesMarker({ type = "line", color, status, visible = true }: ChartSeriesMarkerProps) {
   color = visible ? color : colorTextInteractiveDisabled;
   return (
-    <div className={styles.marker}>
+    <div className={styles.marker} style={{ position: "relative" }}>
```

Comment:
Please move all styles to .scss. We only use inline styles when they need to be computed (or can use in test-pages occasionally).